code/modules/mining/abandoned_crates.dm

```diff
@@ -287,3 +287,48 @@
 		if(25)
 			new /obj/item/weapon/spellbook/oneuse/smoke(src)
 
+
+//Spooky special loot
+
+/obj/item/device/wisp_lantern
+	name = "spooky lantern"
+	desc = "This lantern gives off no light, but is home to a friendly wisp."
+	icon = 'icons/obj/lighting.dmi'
+	icon_state = "lantern-blue"
```

Comment:
If it was a typo, though, this wouldn't work(also I checked the file, not a typo?????)